.github/workflows/ci.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Switch to Xcode 11
-        run: ls /Applications; sudo xcode-select -s /Applications/Xcode_11.5.app
+        run: ls /Applications; sudo xcode-select -s /Applications/Xcode_11.7.app
```

Comment:
What decides the version of Xcode used in this test. And/or what's the justifications for jumping the version here?

Reply:
The current github build machine doesn't have Xcode version 11.5 anymore, so the build was failing before I bumped the version.

You can check the list of currently supported Xcode here: https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#xcode